jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/DefaultServletTest.java

```diff
@@ -2183,7 +2183,7 @@ public static Stream<Arguments> rangeScenarios()
                 String body = response.getContent();
 
                 assertThat(response, containsHeaderValue("Content-Type", "multipart/byteranges"));
-                assertThat(response, containsHeaderValue("Content-Length", "" + body.length()));
+                // TODO #10307 assertThat(response, containsHeaderValue("Content-Length", String.valueOf(body.length())));
```

Comment:
Why were these tests passing before but not now?

Reply:
@lachlan-roberts Because the content was written via the servlet HttpOutput which aggregated them and they were small enough examples to fit into a single buffer.  Thus the content length was known.  Now it is written without the copy, but that means the length is not known when the response is committed.

See issue.